14:53:52.681 [main] DEBUG org.springframework.test.context.junit4.SpringJUnit4ClassRunner - SpringJUnit4ClassRunner constructor called with [class org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.690 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating CacheAwareContextLoaderDelegate from class [org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate]
14:53:52.697 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating BootstrapContext using constructor [public org.springframework.test.context.support.DefaultBootstrapContext(java.lang.Class,org.springframework.test.context.CacheAwareContextLoaderDelegate)]
14:53:52.735 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating TestContextBootstrapper for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests] from class [org.springframework.boot.test.context.SpringBootTestContextBootstrapper]
14:53:52.743 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Neither @ContextConfiguration nor @ContextHierarchy found for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests], using SpringBootContextLoader
14:53:52.747 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/KubernetesDiscoveryClientAutoConfigurationTests-context.xml] does not exist
14:53:52.747 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/KubernetesDiscoveryClientAutoConfigurationTestsContext.groovy] does not exist
14:53:52.748 [main] INFO org.springframework.test.context.support.AbstractContextLoader - Could not detect default resource locations for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: no resource found for suffixes {-context.xml, Context.groovy}.
14:53:52.823 [main] DEBUG org.springframework.test.context.support.ActiveProfilesUtils - Could not find an 'annotation declaring class' for annotation type [org.springframework.test.context.ActiveProfiles] and class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.926 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - @TestExecutionListeners is not present for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]: using defaults.
14:53:52.926 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.event.ApplicationEventsTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener, org.springframework.test.context.event.EventPublishingTestExecutionListener]
14:53:52.937 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.transaction.TransactionalTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/TransactionDefinition]
14:53:52.938 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/interceptor/TransactionAttribute]
14:53:52.938 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@5890e879, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@6440112d, org.springframework.test.context.event.ApplicationEventsTestExecutionListener@31ea9581, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@231f98ef, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@7c137fd5, org.springframework.test.context.support.DirtiesContextTestExecutionListener@183ec003, org.springframework.test.context.event.EventPublishingTestExecutionListener@7d9d0818, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@221a3fa4, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@451001e5, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@2b40ff9c, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@3e08ff24, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@4d1c005e, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener@8462f31]
14:53:52.940 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.940 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.941 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.941 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.952 [main] DEBUG org.springframework.test.context.junit4.SpringJUnit4ClassRunner - SpringJUnit4ClassRunner constructor called with [class org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.953 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating CacheAwareContextLoaderDelegate from class [org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate]
14:53:52.953 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating BootstrapContext using constructor [public org.springframework.test.context.support.DefaultBootstrapContext(java.lang.Class,org.springframework.test.context.CacheAwareContextLoaderDelegate)]
14:53:52.954 [main] DEBUG org.springframework.test.context.BootstrapUtils - Instantiating TestContextBootstrapper for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests] from class [org.springframework.boot.test.context.SpringBootTestContextBootstrapper]
14:53:52.954 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Neither @ContextConfiguration nor @ContextHierarchy found for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests], using SpringBootContextLoader
14:53:52.955 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/reactive/KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests-context.xml] does not exist
14:53:52.956 [main] DEBUG org.springframework.test.context.support.AbstractContextLoader - Did not detect default resource location for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: class path resource [org/springframework/cloud/kubernetes/client/discovery/reactive/KubernetesInformerReactiveDiscoveryClientAutoConfigurationTestsContext.groovy] does not exist
14:53:52.956 [main] INFO org.springframework.test.context.support.AbstractContextLoader - Could not detect default resource locations for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: no resource found for suffixes {-context.xml, Context.groovy}.
14:53:52.959 [main] DEBUG org.springframework.test.context.support.ActiveProfilesUtils - Could not find an 'annotation declaring class' for annotation type [org.springframework.test.context.ActiveProfiles] and class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.981 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - @TestExecutionListeners is not present for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]: using defaults.
14:53:52.981 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Loaded default TestExecutionListener class names from location [META-INF/spring.factories]: [org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener, org.springframework.test.context.web.ServletTestExecutionListener, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener, org.springframework.test.context.event.ApplicationEventsTestExecutionListener, org.springframework.test.context.support.DependencyInjectionTestExecutionListener, org.springframework.test.context.support.DirtiesContextTestExecutionListener, org.springframework.test.context.transaction.TransactionalTestExecutionListener, org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener, org.springframework.test.context.event.EventPublishingTestExecutionListener]
14:53:52.982 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.transaction.TransactionalTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/TransactionDefinition]
14:53:52.982 [main] DEBUG org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Skipping candidate TestExecutionListener [org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener] due to a missing dependency. Specify custom listener classes or make the default listener classes and their required dependencies available. Offending class: [org/springframework/transaction/interceptor/TransactionAttribute]
14:53:52.982 [main] INFO org.springframework.boot.test.context.SpringBootTestContextBootstrapper - Using TestExecutionListeners: [org.springframework.test.context.web.ServletTestExecutionListener@34bde49d, org.springframework.test.context.support.DirtiesContextBeforeModesTestExecutionListener@1b1cfb87, org.springframework.test.context.event.ApplicationEventsTestExecutionListener@821330f, org.springframework.boot.test.mock.mockito.MockitoTestExecutionListener@6f43c82, org.springframework.boot.test.autoconfigure.SpringBootDependencyInjectionTestExecutionListener@5db6b9cd, org.springframework.test.context.support.DirtiesContextTestExecutionListener@210ab13f, org.springframework.test.context.event.EventPublishingTestExecutionListener@20b2475a, org.springframework.boot.test.mock.mockito.ResetMocksTestExecutionListener@7857fe2, org.springframework.boot.test.autoconfigure.restdocs.RestDocsTestExecutionListener@6f15d60e, org.springframework.boot.test.autoconfigure.web.client.MockRestServiceServerResetTestExecutionListener@1be2019a, org.springframework.boot.test.autoconfigure.web.servlet.MockMvcPrintOnlyOnFailureTestExecutionListener@29d80d2b, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverTestExecutionListener@58e1d9d, org.springframework.boot.test.autoconfigure.webservices.client.MockWebServiceServerTestExecutionListener@446a1e84]
14:53:52.982 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.983 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.983 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.983 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.984 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.984 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.985 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.985 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.986 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.987 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.987 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.987 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.987 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.987 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.987 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.988 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.988 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.988 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.988 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.988 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.992 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.993 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.993 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.993 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.994 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.994 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.994 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.994 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.reactive.KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests]
14:53:52.996 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.996 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.996 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:52.996 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:53.000 [main] DEBUG org.springframework.test.context.support.AbstractDirtiesContextTestExecutionListener - Before test class: context [DefaultTestContext@5d908d47 testClass = KubernetesDiscoveryClientAutoConfigurationTests, testInstance = [null], testMethod = [null], testException = [null], mergedContextConfiguration = [WebMergedContextConfiguration@5efa40fe testClass = KubernetesDiscoveryClientAutoConfigurationTests, locations = '{}', classes = '{class org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests$TestConfig}', contextInitializerClasses = '[]', activeProfiles = '{}', propertySourceLocations = '{}', propertySourceProperties = '{spring.cloud.kubernetes.discovery.cacheLoadingTimeoutSeconds=5, spring.cloud.config.enabled=false, spring.cloud.kubernetes.discovery.waitCacheReady=false, org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true, server.port=0}', contextCustomizers = set[org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@662ac478, org.springframework.boot.test.json.DuplicateJsonObjectContextCustomizerFactory$DuplicateJsonObjectContextCustomizer@477b4cdf, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.TestRestTemplateContextCustomizer@367ffa75, org.springframework.boot.test.web.reactive.server.WebTestClientContextCustomizer@1c1bbc4e, org.springframework.boot.test.autoconfigure.actuate.metrics.MetricsExportContextCustomizerFactory$DisableMetricExportContextCustomizer@2aceadd4, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@0, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizerFactory$Customizer@7bedc48a, org.springframework.boot.test.context.SpringBootTestArgs@1, org.springframework.boot.test.context.SpringBootTestWebEnvironment@7722c3c3], resourceBasePath = 'src/main/webapp', contextLoader = 'org.springframework.boot.test.context.SpringBootContextLoader', parent = [null]], attributes = map['org.springframework.test.context.web.ServletTestExecutionListener.activateListener' -> false]], class annotated with @DirtiesContext [false] with mode [null].
14:53:53.001 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved @ProfileValueSourceConfiguration [null] for test class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:53.001 [main] DEBUG org.springframework.test.annotation.ProfileValueUtils - Retrieved ProfileValueSource type [class org.springframework.test.annotation.SystemProfileValueSource] for class [org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests]
14:53:53.005 [main] DEBUG org.springframework.test.context.support.DependencyInjectionTestExecutionListener - Performing dependency injection for test context [[DefaultTestContext@5d908d47 testClass = KubernetesDiscoveryClientAutoConfigurationTests, testInstance = org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests@15d49048, testMethod = [null], testException = [null], mergedContextConfiguration = [WebMergedContextConfiguration@5efa40fe testClass = KubernetesDiscoveryClientAutoConfigurationTests, locations = '{}', classes = '{class org.springframework.cloud.kubernetes.client.discovery.KubernetesDiscoveryClientAutoConfigurationTests$TestConfig}', contextInitializerClasses = '[]', activeProfiles = '{}', propertySourceLocations = '{}', propertySourceProperties = '{spring.cloud.kubernetes.discovery.cacheLoadingTimeoutSeconds=5, spring.cloud.config.enabled=false, spring.cloud.kubernetes.discovery.waitCacheReady=false, org.springframework.boot.test.context.SpringBootTestContextBootstrapper=true, server.port=0}', contextCustomizers = set[org.springframework.boot.test.context.filter.ExcludeFilterContextCustomizer@662ac478, org.springframework.boot.test.json.DuplicateJsonObjectContextCustomizerFactory$DuplicateJsonObjectContextCustomizer@477b4cdf, org.springframework.boot.test.mock.mockito.MockitoContextCustomizer@0, org.springframework.boot.test.web.client.TestRestTemplateContextCustomizer@367ffa75, org.springframework.boot.test.web.reactive.server.WebTestClientContextCustomizer@1c1bbc4e, org.springframework.boot.test.autoconfigure.actuate.metrics.MetricsExportContextCustomizerFactory$DisableMetricExportContextCustomizer@2aceadd4, org.springframework.boot.test.autoconfigure.properties.PropertyMappingContextCustomizer@0, org.springframework.boot.test.autoconfigure.web.servlet.WebDriverContextCustomizerFactory$Customizer@7bedc48a, org.springframework.boot.test.context.SpringBootTestArgs@1, org.springframework.boot.test.context.SpringBootTestWebEnvironment@7722c3c3], resourceBasePath = 'src/main/webapp', contextLoader = 'org.springframework.boot.test.context.SpringBootContextLoader', parent = [null]], attributes = map['org.springframework.test.context.web.ServletTestExecutionListener.activateListener' -> false, 'org.springframework.test.context.event.ApplicationEventsTestExecutionListener.recordApplicationEvents' -> false]]].

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.6.3)

2024-05-17 14:53:53.562  WARN 4105 --- [           main] ubernetesProfileEnvironmentPostProcessor : Not running inside kubernetes. Skipping 'kubernetes' profile activation.
2024-05-17 14:53:53.571  INFO 4105 --- [           main] tesDiscoveryClientAutoConfigurationTests : Starting KubernetesDiscoveryClientAutoConfigurationTests using Java 1.8.0_292 on 9168d08beda2 with PID 4105 (started by idflakies in /home/idflakies/spring-cloud/spring-cloud-kubernetes)
2024-05-17 14:53:53.572  INFO 4105 --- [           main] tesDiscoveryClientAutoConfigurationTests : No active profile set, falling back to default profiles: default
2024-05-17 14:53:54.908  INFO 4105 --- [           main] org.eclipse.jetty.util.log               : Logging initialized @3018ms to org.eclipse.jetty.util.log.Slf4jLog
2024-05-17 14:53:55.040  INFO 4105 --- [           main] org.eclipse.jetty.server.Server          : jetty-9.4.44.v20210927; built: 2021-09-27T23:02:44.612Z; git: 8da83308eeca865e495e53ef315a249d63ba9332; jvm 1.8.0_292-8u292-b10-0ubuntu1~16.04.1-b10
2024-05-17 14:53:55.064  INFO 4105 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@64f981e2{/__admin,null,AVAILABLE}
2024-05-17 14:53:55.066  INFO 4105 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@59bbb974{/,null,AVAILABLE}
2024-05-17 14:53:55.083  INFO 4105 --- [           main] o.e.jetty.server.AbstractConnector       : Started NetworkTrafficServerConnector@7c72ecc{HTTP/1.1, (http/1.1)}{0.0.0.0:41915}
2024-05-17 14:53:55.083  INFO 4105 --- [           main] org.eclipse.jetty.server.Server          : Started @3195ms
2024-05-17 14:53:55.323  INFO 4105 --- [tp2107873140-22] o.e.j.s.handler.ContextHandler.__admin   : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.AdminRequestHandler. Normalized mapped under returned 'null'
2024-05-17 14:53:55.553  INFO 4105 --- [           main] o.s.c.a.ConfigurationClassPostProcessor  : Cannot enhance @Configuration bean definition 'kubernetesDiscoveryClientAutoConfigurationTests.TestConfig' since its singleton instance has been created too early. The typical cause is a non-static @Bean method with a BeanDefinitionRegistryPostProcessor return type: Consider declaring such methods as 'static'.
2024-05-17 14:53:55.662  INFO 4105 --- [           main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=c4d006af-a204-306e-a414-6c511b278167
2024-05-17 14:53:55.757  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:55.792  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration' of type [org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:55.798  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:55.801  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'loadBalancerClientsDefaultsMappingsProvider' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration$$Lambda$446/545392109] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:55.806  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'defaultsBindHandlerAdvisor' of type [org.springframework.cloud.commons.config.DefaultsBindHandlerAdvisor] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:55.811  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'kubernetes.manifests-io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:56.186  INFO 4105 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2024-05-17 14:53:56.199  INFO 4105 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2024-05-17 14:53:56.199  INFO 4105 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.56]
2024-05-17 14:53:56.361  INFO 4105 --- [           main] o.a.c.c.C.[Tomcat].[localhost].[/]       : Initializing Spring embedded WebApplicationContext
2024-05-17 14:53:56.361  INFO 4105 --- [           main] w.s.c.ServletWebServerApplicationContext : Root WebApplicationContext: initialization completed in 2767 ms
2024-05-17 14:53:56.600  INFO 4105 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-05-17 14:53:56.600  INFO 4105 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-05-17 14:53:56.600  INFO 4105 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-05-17 14:53:56.600  INFO 4105 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-05-17 14:53:56.602  INFO 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 14:53:56.602  INFO 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 14:53:56.647  INFO 4105 --- [tp2107873140-15] o.e.j.s.handler.ContextHandler.ROOT      : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.StubRequestHandler. Normalized mapped under returned 'null'
2024-05-17 14:53:56.712 ERROR 4105 --- [tp2107873140-15] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:53:56.712 ERROR 4105 --- [tp2107873140-21] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:53:56.737 ERROR 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:53:56.737 ERROR 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:53:57.598  INFO 4105 --- [pool-9-thread-1] .k.c.d.KubernetesInformerDiscoveryClient : Waiting for the cache of informers to be fully loaded..
2024-05-17 14:53:57.599  INFO 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Cache fully loaded (total 0 services) , discovery client is now available
2024-05-17 14:53:57.739  INFO 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 14:53:57.740  INFO 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 14:53:57.748 ERROR 4105 --- [tp2107873140-15] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:53:57.749 ERROR 4105 --- [tp2107873140-16] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:53:57.750 ERROR 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:53:57.752 ERROR 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:53:58.562  INFO 4105 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 40743 (http) with context path ''
2024-05-17 14:53:58.574  INFO 4105 --- [           main] tesDiscoveryClientAutoConfigurationTests : Started KubernetesDiscoveryClientAutoConfigurationTests in 5.538 seconds (JVM running for 6.685)
2024-05-17 14:53:58.864  INFO 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 14:53:58.864  INFO 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 14:53:58.871 ERROR 4105 --- [tp2107873140-22] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:53:58.872 ERROR 4105 --- [tp2107873140-21] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:53:58.873 ERROR 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:53:58.874 ERROR 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:53:59.139  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Could not find a port named 'oops', 'https', or 'http' for service 'test-svc-1'.
2024-05-17 14:53:59.140  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Make sure that either the primary-port-name label has been added to the service, or that spring.cloud.kubernetes.discovery.primary-port-name has been configured.
2024-05-17 14:53:59.140  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Alternatively name the primary port 'https' or 'http'
2024-05-17 14:53:59.140  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : An incorrect configuration may result in non-deterministic behaviour.
2024-05-17 14:53:59.141  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Could not find a port named 'oops', 'https', or 'http' for service 'test-svc-1'.
2024-05-17 14:53:59.142  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Make sure that either the primary-port-name label has been added to the service, or that spring.cloud.kubernetes.discovery.primary-port-name has been configured.
2024-05-17 14:53:59.142  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Alternatively name the primary port 'https' or 'http'
2024-05-17 14:53:59.142  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : An incorrect configuration may result in non-deterministic behaviour.
2024-05-17 14:53:59.154  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Could not find a port named 'oops', 'https', or 'http' for service 'test-svc-1'.
2024-05-17 14:53:59.154  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Make sure that either the primary-port-name label has been added to the service, or that spring.cloud.kubernetes.discovery.primary-port-name has been configured.
2024-05-17 14:53:59.154  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Alternatively name the primary port 'https' or 'http'
2024-05-17 14:53:59.154  WARN 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : An incorrect configuration may result in non-deterministic behaviour.

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::                (v2.6.3)

2024-05-17 14:53:59.463  WARN 4105 --- [           main] ubernetesProfileEnvironmentPostProcessor : Not running inside kubernetes. Skipping 'kubernetes' profile activation.
2024-05-17 14:53:59.465  INFO 4105 --- [           main] iveDiscoveryClientAutoConfigurationTests : Starting KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests using Java 1.8.0_292 on 9168d08beda2 with PID 4105 (started by idflakies in /home/idflakies/spring-cloud/spring-cloud-kubernetes)
2024-05-17 14:53:59.465  INFO 4105 --- [           main] iveDiscoveryClientAutoConfigurationTests : No active profile set, falling back to default profiles: default
2024-05-17 14:53:59.668  INFO 4105 --- [           main] org.eclipse.jetty.server.Server          : jetty-9.4.44.v20210927; built: 2021-09-27T23:02:44.612Z; git: 8da83308eeca865e495e53ef315a249d63ba9332; jvm 1.8.0_292-8u292-b10-0ubuntu1~16.04.1-b10
2024-05-17 14:53:59.670  INFO 4105 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@7f030c72{/__admin,null,AVAILABLE}
2024-05-17 14:53:59.670  INFO 4105 --- [           main] o.e.jetty.server.handler.ContextHandler  : Started o.e.j.s.ServletContextHandler@68d8ae27{/,null,AVAILABLE}
2024-05-17 14:53:59.672  INFO 4105 --- [           main] o.e.jetty.server.AbstractConnector       : Started NetworkTrafficServerConnector@b808af5{HTTP/1.1, (http/1.1)}{0.0.0.0:39891}
2024-05-17 14:53:59.672  INFO 4105 --- [           main] org.eclipse.jetty.server.Server          : Started @7784ms
2024-05-17 14:53:59.676  INFO 4105 --- [qtp640093047-52] o.e.j.s.handler.ContextHandler.__admin   : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.AdminRequestHandler. Normalized mapped under returned 'null'
2024-05-17 14:53:59.729  INFO 4105 --- [           main] o.s.c.a.ConfigurationClassPostProcessor  : Cannot enhance @Configuration bean definition 'kubernetesInformerReactiveDiscoveryClientAutoConfigurationTests.TestConfig' since its singleton instance has been created too early. The typical cause is a non-static @Bean method with a BeanDefinitionRegistryPostProcessor return type: Consider declaring such methods as 'static'.
2024-05-17 14:53:59.733  INFO 4105 --- [           main] o.s.cloud.context.scope.GenericScope     : BeanFactory id=d7232dc4-7a46-3e4e-9978-4dacfc5f6386
2024-05-17 14:53:59.755  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:59.765  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration' of type [org.springframework.cloud.commons.config.CommonsConfigAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:59.767  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:59.770  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'loadBalancerClientsDefaultsMappingsProvider' of type [org.springframework.cloud.client.loadbalancer.LoadBalancerDefaultMappingsProviderAutoConfiguration$$Lambda$446/545392109] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:59.772  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'defaultsBindHandlerAdvisor' of type [org.springframework.cloud.commons.config.DefaultsBindHandlerAdvisor] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:59.774  INFO 4105 --- [           main] trationDelegate$BeanPostProcessorChecker : Bean 'kubernetes.manifests-io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties' of type [io.kubernetes.client.spring.extended.manifests.config.KubernetesManifestsProperties] is not eligible for getting processed by all BeanPostProcessors (for example: not eligible for auto-proxying)
2024-05-17 14:53:59.846  INFO 4105 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat initialized with port(s): 0 (http)
2024-05-17 14:53:59.847  INFO 4105 --- [           main] o.apache.catalina.core.StandardService   : Starting service [Tomcat]
2024-05-17 14:53:59.847  INFO 4105 --- [           main] org.apache.catalina.core.StandardEngine  : Starting Servlet engine: [Apache Tomcat/9.0.56]
2024-05-17 14:53:59.874  INFO 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 14:53:59.874  INFO 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 14:53:59.881 ERROR 4105 --- [tp2107873140-20] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:53:59.884 ERROR 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:53:59.887 ERROR 4105 --- [tp2107873140-15] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:53:59.889 ERROR 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:54:00.222  INFO 4105 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-05-17 14:54:00.222  INFO 4105 --- [oller-V1Service] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-05-17 14:54:00.222  INFO 4105 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: ready to run resync & reflector runnable
2024-05-17 14:54:00.222  INFO 4105 --- [ler-V1Endpoints] i.k.client.informer.cache.Controller     : informer#Controller: resync skipped due to 0 full resync period
2024-05-17 14:54:00.222  INFO 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 14:54:00.222  INFO 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 14:54:00.225  INFO 4105 --- [qtp640093047-51] o.e.j.s.handler.ContextHandler.ROOT      : RequestHandlerClass from context returned com.github.tomakehurst.wiremock.http.StubRequestHandler. Normalized mapped under returned 'null'
2024-05-17 14:54:00.239 ERROR 4105 --- [qtp640093047-50] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:54:00.241 ERROR 4105 --- [qtp640093047-46] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:54:00.243 ERROR 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:54:00.244 ERROR 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:54:00.884  INFO 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 14:54:00.890 ERROR 4105 --- [tp2107873140-22] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:54:00.890  INFO 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 14:54:00.892 ERROR 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:54:00.896 ERROR 4105 --- [tp2107873140-21] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:54:00.898 ERROR 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:54:01.222  INFO 4105 --- [ool-27-thread-1] .k.c.d.KubernetesInformerDiscoveryClient : Waiting for the cache of informers to be fully loaded..
2024-05-17 14:54:01.223  INFO 4105 --- [           main] .k.c.d.KubernetesInformerDiscoveryClient : Cache fully loaded (total 0 services) , discovery client is now available
2024-05-17 14:54:01.243  INFO 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Start listing and watching...
2024-05-17 14:54:01.245  INFO 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Start listing and watching...
2024-05-17 14:54:01.250 ERROR 4105 --- [qtp640093047-45] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/services?resourceVersion=0&watch=  | /api/v1/namespaces/test/services?watch=true         <<<<< URL does not match
false                                                      |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:54:01.252 ERROR 4105 --- [els.V1Service-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Service#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:54:01.254 ERROR 4105 --- [qtp640093047-50] WireMock                                 : 
                                               Request was not matched
                                               =======================

-----------------------------------------------------------------------------------------------------------------------
| Closest stub                                             | Request                                                  |
-----------------------------------------------------------------------------------------------------------------------
                                                           |
GET                                                        | GET
/api/v1/namespaces/test/endpoints?resourceVersion=0&watch  | /api/v1/namespaces/test/endpoints?watch=true        <<<<< URL does not match
=false                                                     |
                                                           |
                                                           |
-----------------------------------------------------------------------------------------------------------------------

2024-05-17 14:54:01.257 ERROR 4105 --- [s.V1Endpoints-1] i.k.c.informer.cache.ReflectorRunnable   : class io.kubernetes.client.openapi.models.V1Endpoints#Reflector loop failed unexpectedly

io.kubernetes.client.openapi.ApiException: Not Found
	at io.kubernetes.client.util.Watch.createWatch(Watch.java:105) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:761) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.util.generic.GenericKubernetesApi.watch(GenericKubernetesApi.java:281) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.SharedInformerFactory$2.watch(SharedInformerFactory.java:263) ~[client-java-13.0.0.jar:na]
	at io.kubernetes.client.informer.cache.ReflectorRunnable.run(ReflectorRunnable.java:120) ~[client-java-13.0.0.jar:na]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) ~[na:1.8.0_292]
	at java.util.concurrent.FutureTask.runAndReset(FutureTask.java:308) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.access$301(ScheduledThreadPoolExecutor.java:180) ~[na:1.8.0_292]
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:294) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149) ~[na:1.8.0_292]
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624) ~[na:1.8.0_292]
	at java.lang.Thread.run(Thread.java:748) ~[na:1.8.0_292]

2024-05-17 14:54:01.447  INFO 4105 --- [           main] o.s.b.w.embedded.tomcat.TomcatWebServer  : Tomcat started on port(s): 33313 (http) with context path ''
2024-05-17 14:54:01.453  INFO 4105 --- [           main] iveDiscoveryClientAutoConfigurationTests : Started KubernetesInformerReactiveDiscoveryClientAutoConfigurationTests in 2.032 seconds (JVM running for 9.564)
